io::CharsError not found

error[E0412]: cannot find type `CharsError` in module `io`
  --> src/error.rs:16:24
   |
16 |         Chars(err: io::CharsError) {
   |                        ^^^^^^^^^^ not found in `io`

error: aborting due to previous error

For more information about this error, try `rustc --explain E0412`.
error: Could not compile `inquirer`.

To learn more, run the command again with --verbose.

--- a/src/error.rs
+++ b/src/error.rs
@@ -11,14 +11,6 @@ quick_error! {
             display("I/O error")
             description(err.description())
         }
-        /// Input read error
-        // `std::io::CharsError` is unstable!
-        Chars(err: io::CharsError) {
-            from()
-            cause(err)
-            display("Chars error")
-            description(err.description())
-        }
         /// Invalid choice
         // TODO: Make this a type system error instead
         InvalidChoice(option_num: usize) {

--- a/src/lib.rs
+++ b/src/lib.rs
@@ -1,8 +1,5 @@
 //! Inquirer – Fancy user interaction on the command line
 
-// Termion depends on nighty (for Keys), we need this for the unstable `std::io::CharsError`
-#![feature(io)]
-
 #![deny(missing_docs,
     missing_debug_implementations, missing_copy_implementations,
     trivial_casts, trivial_numeric_casts,

